sky/cli.py

```diff
@@ -1654,6 +1660,10 @@ def status(all: bool, refresh: bool, show_spot_jobs: bool, clusters: List[str]):
 
     Display all fields using ``sky status -a``.
 
+    Only display the IP address of the cluster using
+    ``sky status --ip <cluster-name>``. This option will override all other
+    options.
```

Comment:
Should we add a warning for Kubernetes:
```suggestion
    options. For Kubernetes clusters, the returned IP address is the internal IP of the head pod, and may not be accessible from outside the cluster. 
```